dietpi/dietpi-software

```diff
@@ -1514,6 +1514,24 @@ _EOF_
 	   aSOFTWARE_REQUIRES_JAVA_JRE_JDK[$index_current]=1
 			 aSOFTWARE_ONLINEDOC_URL[$index_current]='f=8&t=5&p=6202#p6202'
 
```

Comment:
@Fourdee 
Watching alignment of the software arrays, I guess your local code editor handles tabs differently than github (tab = 8 digits space)?
Found my notepad++ with default 2 spaces, if I remember right, so raw downloads from github looked wrong at first.

Shall we align all of them to look pretty on github... by times... ?

Reply:
@MichaIng 

I use notepad++ with default tab size of 4
![image](https://user-images.githubusercontent.com/7077026/43685002-f37c32a6-98a2-11e8-86a5-ece1666b20e6.png)

![image](https://user-images.githubusercontent.com/7077026/43685010-0ad416da-98a3-11e8-91f4-fa6f90f491b7.png)

> Shall we align all of them to look pretty on github... by times... ?

Yep, we probably should do that (at some point, minor). Apparently GitHub default is 8 for a tab:
https://github.com/lukechilds/github-custom-tab-size#github-custom-tab-size

Reply:
@Fourdee 
Jep, it depends on the file ending as well, AFAIK. For .txt/.md files I remember a default of `<tab>` being translated into 2 times `<blank>`. But at least all our script files have tab -> tab with 8 blank width.